source/endpoints/devices/Activator.py

```diff
@@ -0,0 +1,41 @@
+from flask_restplus import Resource
+from flask_restplus import fields
+
```

Comment:
This is python convention as far as I understand. To highlight the difference between internal and external imports.